src/components/PieChart/index.jsx

```diff
@@ -4,7 +4,7 @@ import generateRandomColor from "../../shared/colorsGenerate";
 
 const PieChart = ({ data }) => {
     const colors = generateRandomColor();
```

Comment:
How you used this function without a parameter?
![image](https://user-images.githubusercontent.com/74979055/107661391-ae515680-6ca2-11eb-9fc8-1dc1753e2e95.png)